src/kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -1004,7 +1013,7 @@ impl<'tcx> GotocCtx<'tcx> {
                 // Call this function recursively to compute the size and align for the last field.
                 let field_ty = layout.field(self, n).ty;
                 let SizeAlign { size: unsized_size, align: mut unsized_align } =
-                    self.size_and_align_of_dst(field_ty, arg);
+                    self.size_and_align_of_dst(field_ty, arg.clone())?;
```

Comment:
Why is this clone now needed?

Reply:
This was probably some debug I removed and forgot to revert this. I'll remove it.